lib/src/components/NxDropdown/NxDropdown.tsx

```diff
@@ -6,16 +6,14 @@
  */
 import React, { FunctionComponent } from 'react';
 import classnames from 'classnames';
-import { faCaretDown } from '@fortawesome/free-solid-svg-icons';
 
 import { Props, propTypes } from './types';
 import NxTooltip from '../NxTooltip/NxTooltip';
 import NxButton from '../NxButton/NxButton';
-import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
 import { wrapTooltipProps } from '../../util/tooltipUtils';
-import { ensureElement } from '../../util/reactUtil';
-
 import './NxDropdown.scss';
+import {faCaretDown, faCaretUp} from '@fortawesome/free-solid-svg-icons';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
```

Comment:
Minor: reorganize the imports such that third party imports are in a section at the top and then local imports are in the next section, followed by the scss import.  i.e, put these two back where they were before this PR.

Reply:
> Minor: reorganize the imports such that third party imports are in a section at the top and then local imports are in the next section, followed by the scss import. i.e, put these two back where they were before this PR.

https://github.com/sonatype/sonatype-react-shared-components/pull/55/commits/e4520e36cddcb0667a07dc21e922242c20c1c424